tests/test_request.py

```diff
@@ -1603,9 +1602,14 @@ def test_encget_with_encattr(self):
             val = b'\xc3\xab'.decode('latin-1')
         else:
             val = b'\xc3\xab'
-        inst = self._makeOne({'a':val})
+        inst = self._makeOne({'a': val})
         assert inst.encget('a', encattr='url_encoding') == text_(b'\xc3\xab', 'utf-8')
 
+    def test_encget_does_raises_on_invalid_utf_8(self):
```

Comment:
What do you mean? It tests that if we provide a invalid utf-8 character in the PATH it ignores it right?